src/uploader.ts

```diff
@@ -89,12 +89,12 @@ export default class Uploader {
       upload = ajax.transport({
         url: this.config.endpoints.byFile,
         data: this.config.additionalRequestData,
-        accept: this.config.types,
+        accept: this.config.types || 'image/*',
         headers: new Headers(this.config.additionalRequestHeaders as Record<string, string>),
         beforeSend: (files: File[]) => {
           preparePreview(files[0]);
         },
-        fieldName: this.config.field,
```

Comment:
so we should remove the same from uploader.ts? since it will never called

Reply:
what should be removed?

Reply:
sorry, confused